frappe/utils/background_jobs.py

```diff
@@ -18,12 +18,17 @@
 from frappe import _
 from frappe.utils import cstr
 
+site_config = frappe.get_site_config()
+common_site_config = frappe.get_file_json("common_site_config.json")
+custom_workers_config = site_config.get("workers", common_site_config.get("workers", {}))
```

Comment:
Also, workers are a bench level concept. So their config should exist in the common_site_cofnig file alone? Just checking `custom_workers_config = common_site_config.get("workers", {})` should work.